Configurations/config.template.js

```diff
@@ -1,161 +1,158 @@
+	 */
+	hostingURL: "https://dev.gawesomebot.com/",
+	/*
+	 * The server IP the web inteface should point itself to.
+	 * Preferably, you leave it as *0.0.0.0* so that it is on the local machine
+	 */
+	serverIP: "0.0.0.0",
+	/*
+	 * The HTTP / HTTPS ports the bot should use. If you use Cloudflare, you should just keep httpPort to *80*
+	 * They are both Integers
+	 */
+	httpPort: 80,
+	httpsPort: 433,
+	/*
+	 * Certificate / Private Key paths to the files, required for HTTPS.
+	 * If you are on Windows, make sure to escape the *\*
+	 */
+	cert: "",
+	privateKey: "",
+	/*
+	 * If the bot should try to automatically redirect to https.
+	 * Needs cert and privateKey to be fulfilled
+	 */
+	httpsRedirect: false,
+	/*
+	 * The database connection options
+	 */
+	database: {
+		/*
+		 * The URL pointing to the MongoDB instance GAB should use
+		 */
+		URL: "mongodb://localhost:27017/",
+		/*
+		 * The name of the Database GAB should use
+		 */
+		db: "gabbe",
+	},
+	/*
+	 * MaxVoiceChannels should remain as is.
+	 */
+	maxVoiceChannels: 2,
+	/*
+	 * Direct Discord link to invite your bot. You can replace the permissions, if you know what you are doing.
+	 * Otherwise. !! DO NOT TOUCH !!
+	 */
+	oauthLink: "https://discordapp.com/oauth2/authorize?&client_id={id}&scope=bot&permissions=470281471",
+	/*
+	 * Level of output the console should display. List of levels:
+	 * error, warn, info, debug and verbose.
+	 * Recommended to stay as info to prevent console cluttering.
+	 * fileLevel determines the output to master/shard.gawesomebot.log and verbose.gawesomebot.log files.
+	 * Note that setting these values to a level lower than `info` might influence your bot's performance.
+	 */
+	consoleLevel: "info",
+	fileLevel: "info",
+	/*
+	 * The secret used to sign session hashes.
+	 * This must be a random and secure string!
+	 * Leaving this as default is NOT recommended.
+	 */
+	secret: "vFEvmrQl811q2E8CZelg4438l9YFwAYd",
+	/**
+	 * A secure, random password used to separate your main encryption key from
+	 * someone elses.
+	 * YOU SHOULD CHANGE THIS.
+	 * And DO NOT CHANGE IT AFTERWARDS.
+	 * Use https://passwordsgenerator.net/ to generate something big and secure!
+	 */
+	encryptionPassword: "fEmH!bRP4S8a4^*fpQ=%",
+	/**
+	 * The initial value that will be used in the encryption of data
+	 * YOU MUST CHANGE THIS.
+	 * And whatever you do. DO NOT CHANGE THIS AFTERWARDS.
+	 * The encrypted data will be lost otherwise.
+	 * So, set it to something secore, then never ever touch it. Save it in a file!
+	 * Hell, it can even be a random word! Just remember: never share this.
+	 * Make sure it isn't longer than 16 characters!
+	 * You can use https://passwordsgenerator.net/ to generate an encryption iv.
+	 */
+	encryptionIv: "usM9^y5KHtF_UCn?",
+	/*
+	 * Discord channel and server where any errors should be outputted,
+	 * whenever applicable
+	 * debugChannel: The Channel ID in the server
+	 */
+	discord: {
+		debugChannel: ``,
```

Comment:
Is this still used?

Reply:
It was never used in the first place iirc.

Reply:
Check usages, then remove, I’d say.

Reply:
Was planned to be used, feel free to remove for now, I'll re-add if we're ever implementing it